src/components/project/ProjectCard.tsx

```diff
@@ -35,6 +38,11 @@ const ProjectCard = ({ project, renderAsLink, showActions }: IProps): JSX.Elemen
     uuid,
   } = project;
 
+  const timeNow = new Date().getTime();
+  const applicationPeriodHasStarted = timeNow > new Date(application_start_time).getTime();
+  const applicationPeriodHasEnded = timeNow > new Date(application_end_time).getTime();
+  const canStartLottery = applicationPeriodHasStarted && applicationPeriodHasEnded;
```

Comment:
Does this logic contain an error or am I missing something... how could an application period have ended but not started? 🙂

Reply:
I added this here to prevent human errors, and in the case that either one of the values is missing. They are not required fields in the backend so better be safe.

Reply:
Indeed, this had an issue with null date values, which is now fixed.